config.js

```diff
@@ -50,6 +50,9 @@ const iplist = configUtil.parseIPlist(ipWhitelist);
 // Check for timed-out samples every 30 seconds if not specified in env var
 const DEFAULT_CHECK_TIMEOUT_INTERVAL_MILLIS = 30000;
 
+// GET Samples Cache invalidation default time
+const GET_SAMPLES_WILDCARD_CACHE_INVALIDATION = 5;
```

Comment:
call this DEFAULT_GET_SAMPLES_WILDCARD_CACHE_INVALIDATION

Reply:
ok